lib/ddr/models/has_admin_metadata.rb

```diff
@@ -1,78 +1,77 @@
 require "resque"
 
-module Ddr
-  module Models
-    module HasAdminMetadata
```

Comment:
Only significant change is adding `:depositor` attribute.  The rest is spacing due to collapsing Ddr::Models modules.